### ISSUE TITLE
What happen?

### ISSUE THREAD

**Issue by GF-Huang** [Foundry376/Mailspring#2448]
![image](https://user-images.githubusercontent.com/4510984/215391435-2afa1ad5-a77d-4a21-8018-c2d597e26d0b.png)

MS Edge 109.0.1518.70

**Comment by andrewthiesen** [comment]
@GF-Huang - This looks like this page is being blocked by Microsoft Defender Smartscreen. Most likely, this is because Mailspring downloads as a `.zip` file. As ZIP files can contain _potentially_ malicious software, the Defender Smartscreen settings are probably blocking this download. 

I was able to download successfully on Mac, Linux, and Windows 10 (with Defender disabled).

**Comment by Phylu** [comment]
Duplicate of https://github.com/Foundry376/Mailspring/issues/2451. Should be resolved by now.
